<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorListWindow</class>
 <widget class="QDockWidget" name="ColorListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>585</height>
   </rect>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="windowTitle">
    <string>Dialog</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="newButton">
          <property name="text">
           <string>New</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="color_widgets::ColorWheel" name="wheel" native="true">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>50</y>
            <width>308</width>
            <height>544</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="0">
            <widget class="QLabel" name="label_green">
             <property name="text">
              <string>Green</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Hex ARGB</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lable_alpha">
             <property name="text">
              <string>Alpha</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Color Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QSpinBox" name="spin_alpha"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_blue">
             <property name="text">
              <string>Blue</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="color_widgets::GradientSlider" name="slider_alpha">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QSpinBox" name="spin_green"/>
           </item>
           <item row="4" column="2">
            <widget class="QSpinBox" name="spin_blue"/>
           </item>
           <item row="9" column="1" colspan="2">
            <widget class="QLineEdit" name="value"/>
           </item>
           <item row="2" column="2">
            <widget class="QSpinBox" name="spin_red"/>
           </item>
           <item row="2" column="1">
            <widget class="color_widgets::GradientSlider" name="slider_red">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="color_widgets::GradientSlider" name="slider_green">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_red">
             <property name="text">
              <string>Red</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QSpinBox" name="spinBox"/>
           </item>
           <item row="10" column="1" colspan="2">
            <widget class="QLineEdit" name="name"/>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="color_widgets::GradientSlider" name="slider_blue">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="3">
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>color_widgets::ColorWheel</class>
   <extends>QWidget</extends>
   <header>color_wheel.hpp</header>
   <container>1</container>
   <slots>
    <signal>colorSelected(QColor)</signal>
    <slot>setColor(QColor)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>color_widgets::GradientSlider</class>
   <extends>QSlider</extends>
   <header>gradient_slider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>spin_red</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_red</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_green</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_green</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_blue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_blue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spin_alpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>slider_alpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
